### PR TITLE
added version check warning for all commands but 'generate'

### LINKED ISSUE #721: Add a message in the CLI to update Kusk CLI
```bash
$ kusk deploy -i openapi.yaml
New version of the Kusk CLI is released: use xxxx to upgrade the CLI

API deployed
```

### PR DESCRIPTION
**Pull request by jasmingacic**
Signed-off-by: jasmingacic <jasmin.gacic@gmail.com>

Fixes #721 

image

### CODE REVIEW

**Review by aabedraba — COMMENTED** [review]
BEAUTIFUL message!

File: cmd/kusk/cmd/root.go [hunk 1]
```diff
@@ -44,6 +49,35 @@ var rootCmd = &cobra.Command{
...
+						errors.NewErrorReporter(cmd, err).Report()
+					}
+
+					currentVersion, err := version.NewVersion(build.Version)
+					if err != nil {
+						errors.NewErrorReporter(cmd, err).Report()
+					}
+
+					if currentVersion.LessThan(latestVersion) {
+						ui.Warn(fmt.Sprintf("This version %s of Kusk cli is outdated. The latest version available is %s\n", currentVersion, latestVersion), "Please follow instructions to update you installation: https://docs.kusk.io/getting-started#1-install-kusk-cli ")
```
Comment (aabedraba) [inline cmd/kusk/cmd/root.go:76]:
```suggestion
						ui.Warn(fmt.Sprintf("This version %s of Kusk cli is outdated. The latest version available is %s\n", currentVersion, latestVersion), "Please follow instructions to update you installation: hhttps://docs.kusk.io/reference/cli/overview#updating ")
```

Comment (jasmingacic) [reply cmd/kusk/cmd/root.go]: Done

**Review by mbana — APPROVED** [review]
LGTM.

**Pull request merged**